BDInterface/data_cleaning_functions.R

```diff
@@ -185,4 +185,33 @@ check_for_mixed_polarity <- function(combined_data){
   combined_data <- mutate(combined_data, con_type=ifelse(max_power > ac * .1 & min_power * -1 > ac * 0.1, 
                                                          "mixed", con_type))
   return(combined_data)
-}
\ No newline at end of file
+}
```

Comment:
@phoebeheywood the following addresses low voltage concerns by setting permanent outliers to NaN